cheroot/test/test_core.py

```diff
@@ -16,8 +17,12 @@
 
 
 localDir = os.path.dirname(__file__)
-favicon_path = os.path.join(os.getcwd(), localDir, '../favicon.ico')
```

Comment:
You are mixing `os.path` methods in, while having imported pathlib you could've reused its features of path construction